docs/README.md

```diff
@@ -0,0 +1,10 @@
+# Linode API Developer Docs v4
+
+## Instructions
+1. `cd` to `docs/`
+1. `./transform.sh`
```

Comment:
want to add a section below this for the conversion scripts, and add a short blurb on what ./transform does, then i'll add a blurb on the other after